<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemBreakpointScriptDialog</class>
 <widget class="QDialog" name="SystemBreakpointScriptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System breakpoint scripts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/icons/images/initscript.png</normaloff>:/icons/images/initscript.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxGlobal">
     <property name="title">
      <string>1. System breakpoint script for every process</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelGlobal">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditGlobal"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGlobal">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxDebuggee">
     <property name="title">
      <string>2. System breakpoint script for a specific process (debug a process to specify)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="labelDebuggee">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDebuggee"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDebuggee">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SystemBreakpointScriptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SystemBreakpointScriptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
